yb-voyager/cmd/root.go

```diff
@@ -72,7 +72,7 @@ func init() {
 		"enable verbose mode for the console output")
 
 	rootCmd.PersistentFlags().StringVarP(&exportDir, "export-dir", "e", "",
-		"export directory (default is current working directory")
+		"export directory (default is current working directory)")
```

Comment:
`--export-dir` is a required argument. Required arguments never have default values.

Reply:
done, updated the help message.